databroker/v1.py

```diff
@@ -1640,14 +1640,29 @@ def _from_v0_config(config, auto_register, name):
         dotted_object = '.'.join((contents['module'], contents['class']))
         handler_registry[spec] = dotted_object
     root_map = config.get('root_map')
+
+    # Get transforms.
+    if not transforms:
+        transforms = _load_transforms(config.get('transforms'))
+
     return BlueskyMongoCatalog(metadatastore_db, asset_registry_db,
                                handler_registry=handler_registry,
                                root_map=root_map,
-                               name=name)
+                               name=name,
+                               transforms=transforms)
+
 
 _mongo_clients = {}  # cache of pymongo.MongoClient instances
 
 
+def _load_transforms(transforms):
```

Comment:
If we reuse the same pattern that we use for handlers, these would have a form like `package.subpackage.module.class`. See https://github.com/bluesky/databroker/blob/521f87c0087084cf7a23f6a2c24df06b512169cd/databroker/core.py#L1418-L1443 for an example of how to parse that.